internal/appconfig/machines.go

```diff
@@ -49,6 +49,36 @@ func (c *Config) ToReleaseMachineConfig() (*api.MachineConfig, error) {
 	return mConfig, nil
 }
 
+func (c *Config) ToConsoleMachineConfig() (*api.MachineConfig, error) {
+	mConfig := &api.MachineConfig{
+		Init: api.MachineInit{
+			// TODO: this is an ugly hack, abusing Hallpass as a
+			// stand-in for `/bin/sleep infinity`, simply because it
+			// should always be available.
+			Cmd:        []string{"dummy", "to", "keep", "machine", "running"}, // just to clear the CMD; Hallpass ignores it
+			Entrypoint: []string{"/.fly/hallpass"},
+		},
+		Restart: api.MachineRestart{
+			Policy: api.MachineRestartPolicyNo,
+		},
+		AutoDestroy: true,
+		DNS: &api.DNSConfig{
+			SkipRegistration: true,
+		},
+		// TODO: should we set metadata on these machines?
```

Comment:
Yes! setting the process group and platform is particularly important. Also ensuring those machines are not part of fly deploy.